api/app/models/theq/permission.py

```diff
@@ -12,9 +12,9 @@
 See the License for the specific language governing permissions and
 limitations under the License.'''
 
-
 from qsystem import db
-from .base import Base
+from app.models.theq import Base
+#from app.models import Role
```

Comment:
What's this guy doing here?

Reply:
Un commented this, and added the correct directory after models. Done